lib/resources/postgres_conf.rb

```diff
@@ -74,7 +74,10 @@ def read_content
         raw_conf = read_file(to_read[0])
         @content += raw_conf
 
-        params = SimpleConfig.new(raw_conf).params
+        opts = {
+          assignment_re: /^\s*([^=]*?)\s*=\s*[']*\s*(.*?)\s*[']*\s*$/,
```

Comment:
Haven't seen an empty assignment, but we should enable this. I'll update the unit tests accordingly.